Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2384,6 +2384,15 @@ void RenderObject::RepaintRects::transform(const TransformationMatrix& matrix, f
         *outlineBoundsRect = LayoutRect(encloseRectToDevicePixels(matrix.mapRect(*outlineBoundsRect), deviceScaleFactor));
 }
 
+PointerEvents RenderObject::usedPointerEvents() const
+{
+    // FIXME: This could be a performance problem, checking this all
+    // the time, when view-transitions are rare.
+    if (document().renderingSuppressedForViewTransition() && !isDocumentElementRenderer())
+        return PointerEvents::None;
+    return style().usedPointerEvents();
```

Comment:
There are many style().usedPointerEvents(); calls in WebCore (many of them in some svg code). How do you tell if a newly added call should consult the renderer or the renderer's style? maybe it's always the former? -but then this PR only changes 2 of those callsites. is that sufficient?